src/pages/Compare/index.tsx

```diff
@@ -194,6 +203,7 @@ export class CompareBase extends React.Component<Props> {
         </Helmet>
         <VersionFileViewer
           compareInfo={compareInfo}
+          comparedToVersionId={comparedToVersionId}
```

Comment:
This needs some test coverage since the value is derived from `baseVersionId` in the URL.